independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/CreateProjectHelper.java

```diff
@@ -78,10 +82,16 @@ public static void setJavaVersion(Map<String, Object> values, String javaTarget)
 
         Matcher matcher = JAVA_VERSION_PATTERN
                 .matcher(javaTarget != null ? javaTarget : System.getProperty("java.version", ""));
-        if (matcher.matches() && Integer.parseInt(matcher.group(1)) < 11) {
-            values.put(ProjectGenerator.JAVA_TARGET, "8");
+
+        if (matcher.matches()) {
+            int versionExtracted = Integer.parseInt(matcher.group(1));
+            int version = JAVA_VERSIONS_LTS.stream()
+                    .filter(e -> e.equals(versionExtracted))
+                    .findFirst().orElse(DEFAULT_JAVA_VERSION);
+
+            values.put(ProjectGenerator.JAVA_TARGET, String.valueOf(version));
```

Comment:
In this case it's easier to deal with `String` directly instead of `Integer` (avoids parsing to int and formatting back to String)

Reply:
do we not need the “anything less than 11 means 8” logic anymore?

Reply:
(do we support creating projects for 8 anymore? As of Quarkus 2, we’ve moved on to Java 11… is that the floor?)

Reply:
@gastaldi @ebullient,thanks for the appointments
Fixs done.

Reply:
> (do we support creating projects for 8 anymore? As of Quarkus 2, we’ve moved on to Java 11… is that the floor?)

Yes, Java 11 is the minimum. Java 8 is not supported anymore

Reply:
i removed the possibility to set java 8